presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -346,7 +346,7 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateTask(
   for (const auto& source : sources) {
     // Add all splits from the source to the task.
     LOG(INFO) << "Adding " << source.splits.size() << " splits to " << taskId
-              << " for " << source.planNodeId;
+              << " for node " << source.planNodeId;
```

Comment:
```
nit: s/node/plan node/ be more specific? Thanks!
```

Reply:
Initially I had it, but decided to keep it shorter to reduce the # of characters in the logs. :)
For debugging purposes it is clear enough, imho.